### PR TITLE
upgrade apt version to 1.8.2.1 in debian-base image case by CVE-2020-3810

### PR DESCRIPTION
**Pull request by Zheaoli**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

/kind release
/area security

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #https://github.com/kubernetes/kubernetes/issues/93114

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
upgrade apt version to 1.8.2.1 in debian-base image case by CVE-2020-3810 
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by BenTheElder** [comment]
@kubernetes/sig-release 
not sure if this is something we want in 1.19

File: build/dependencies.yaml [hunk 1]
```diff
@@ -77,13 +77,13 @@ dependencies:
       match: GOLANG_VERSION
 
   - name: "k8s.gcr.io/debian-base"
-    version: 2.1.0
+    version: 2.1.1
```
Comment (BenTheElder) [inline build/dependencies.yaml:80]: this has to happen _after_ the image exists.

Comment (justaugustus) [reply build/dependencies.yaml:80]: This version should be incremented to trigger the build, but the `dependents` version cannot be updated until the image is promoted.

**Comment by justaugustus** [comment]
@Zheaoli -- I've opened https://github.com/kubernetes/kubernetes/pull/93562, which carries your commit and fixes the versions in build/dependencies.yaml.

/close

**Comment by Zheaoli** [comment]
thanks @justaugustus

**Pull request closed without merging**